docs/hub/model-cards.md

```diff
@@ -1,5 +1,6 @@
 # Model Cards
 
+  ### <Tip>[ New! Try our experimental Modal Card Creator App](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)</Tip>
```

Comment:
```suggestion
<Tip>
[New! Try our experimental Model Card Creator App](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)
</Tip>
```

Reply:
I can't seem to comment on your comment below but it looks good to me 
thanks, @osanseviero and @julien-c for your inputs 🤗